### DISCUSSION TITLE
How do i config the java thing.

### DISCUSSION THREAD

**Discussion by Nightazl** [jagrosh/MusicBot#1090] [Questions] [answered]
I downloaded the java thing and the source code but I don't know how to config the commands.

**Comment by YJSoft** [comment, marked as answer]
You don't need source code to use JMusicBot. Please read [wiki](https://jmusicbot.com/setup/) for how to setup.

Also you can refer below pages.
[Example config](https://jmusicbot.com/config/) - See comments for details
[Advanced config options](https://jmusicbot.com/advanced-config/)
